packages/mds-agency/request-handlers.ts

```diff
@@ -254,18 +283,16 @@ export const submitVehicleEvent = async (req: AgencyApiRequest, res: AgencyApiRe
     if (message.includes('duplicate')) {
       logger.info(name, 'duplicate event', event.event_type)
       res.status(409).send({
-        error: 'duplicate_event',
-        error_description: 'an event with this device_id and timestamp has already been received'
+        error: new ConflictError('an event with this device_id and timestamp has already been received')
       })
     } else if (message.includes('not found') || message.includes('unregistered')) {
       logger.info(name, 'event for unregistered', event.device_id, event.event_type)
       res.status(400).send({
-        error: 'unregistered',
-        error_description: 'the specified device_id has not been registered'
+        error: new DependencyMissingError('the specified device_id has not been registered')
```

Comment:
Need to retain the format of `error` and `error_description` from the [Agency Events](https://github.com/openmobilityfoundation/mobility-data-specification/tree/dev/agency#vehicle---event) spec.